FSW/src/util/comUtil.cpp

```diff
@@ -207,23 +207,28 @@ void executeCommand(int command)
     {
         // Mode change
         case ENTER_SAFE_MODE: 
-            // TODO: call corresponding function in timing module
+            scienceMode.set(SAFE_MODE);
+            Serial.println("Command Recieved - Entering Safe Mode. This mode must be manually exited.");
             break;
         
         case ENTER_STANDBY_MODE: 
-            // TODO: call corresponding function in timing module
+            scienceMode.set(STANDBY_MODE);
+            Serial.println("Command Recieved - Entering Standby Mode. This mode must be manually exited.");
```

Comment:
Is there an intermediate mode between this standby mode and a data collection mode? If a manual command is set to leave standby mode, what mode is automatically entered. Does it depend on the location of orbit?

Reply:
There is no "leave standby mode" command, only commands to send the payload to one of the other states related to data gathering. `SUNSET_MODE` will wait for the photodiode output to drop to zero then save the data, `SUNRISE_MODE` saves the buffer after a timer, etc.